cmd/oci-trace-hook/trace.go

```diff
@@ -0,0 +1,332 @@
+// +build oci_trace_hook
+
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/binary"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/signal"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"syscall"
+
+	"github.com/docker/docker/api/types"
+	"github.com/iovisor/gobpf/bcc" //nolint
+	spec "github.com/opencontainers/runtime-spec/specs-go"
+	seccomp "github.com/seccomp/libseccomp-golang"
+	"github.com/sirupsen/logrus"
+)
+
+// event struct used to read data from the perf ring buffer from the
```

Comment:
nit "from the"?  what?

Reply:
Thanks for pointing out the error, fixed :D